edx_event_bus_kafka/publishing/test_event_producer.py

```diff
@@ -65,13 +65,14 @@ def test_get_producer_for_signal(self):
             assert ep.get_producer_for_signal(signal, 'user.id') is None
             assert len(caught_warnings) == 1
             assert str(caught_warnings[0].message).startswith("Cannot configure event-bus-kafka: Missing setting ")
+            print (caught_warnings[0].message)
```

Comment:
There's a debugging line here that should get removed.

Reply:
will do